MDS_3/test/MDS_3/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Created by the script cgal_create_cmake_script
+# This is the CMake script for compiling a CGAL application.
+
+cmake_minimum_required(VERSION 3.1...3.15)
+project( MDS_3_Tests )
+
+if(POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
+endif()
```

Comment:
`CMP0074` was new in CMake-3.12. It is useless, because `cmake_minimum_required(VERSION 3.1...3.15)` already declares that all policies before CMake-3.15 are assumed having the `NEW` behavior.
```suggestion
```